SR2023/2023-03-15-competition-details..md

```diff
@@ -0,0 +1,56 @@
+---
+to: Student Robotics 2023
+subject: SR2023 Competition Details
+---
+
+Hello all!
+
+It's only 2 weeks until the big day - [the competition](https://studentrobotics.org/events/sr2023/competition/)!
+
+Before then, we have a few important updates and reminders for you:
+
+## Media Consent
+
+Throughout the competition, Student Robotics volunteers will be taking photos and videos for the purpose of prompting and marketing the charity. The arena will also be [livestreamed](https://studentrobotics.org/events/sr2023/competition/#livestream) on our YouTube channel. If this is a problem for you or your competitors, please let our reception desk know when you arrive.
+
+The competition this year is a public event, and we do not have exclusive access to the venue.
+
+## Equipment
+
+Please ensure that your team bring at least 1 laptop or appropriate device with them to the competition to access the docs and IDE on. We are unable to provide any computing devices for competitor use at the competition. We will be providing a wireless internet connection to all teams.
+
+Any tools that your team requires to modify their robot must be brought to the competition by the team: we cannot provide any. In particular, the use of any dangerous tools must be supervised by the responsible adult for the team. Power tools may only be used in the designated "Power Tools" area.
+
+You must bring all of the robotics kit (including the white box it came in) to the competition, so that we can collect it at the end of the event, unless you've already arranged to keep the kit longer.
```

Comment:
Are we doing kit extensions? (I had thought not). If not: we should drop the last phrase here.